src/pages/patientView/PatientViewUrlWrapper.ts

```diff
@@ -47,4 +51,15 @@ export default class PatientViewUrlWrapper extends URLWrapper<
     public setTab(tab: string): void {
         this.updateURL({}, `${PagePath.Patient}/${tab}`);
     }
+
+    @computed public get tabId() {
```

Comment:
activeTabId? 
activeSecondaryTabId?

Reply:
not sure what you mean

Reply:
I mean, tabId is maybe not quite descritive enough.  Active tells me that this is the tab that is currently showing.  Sorry if I misundersatnd.

Reply:
activeTabId makes sense. I dont get the secondary part though